cmd/demo/validator.go

```diff
@@ -93,3 +94,16 @@ func nonce() *big.Int {
 	}
 	return val
 }
+
+// etherToWei converts amount in "ether" (represented as float) to "wei" (represented as integer).
+// It can provide exact results for values in the range of 1e-18 to 1e9.
+func etherToWei(ether *big.Float) *big.Int {
```

Comment:
change to variadic function, better usability.

Reply:
Addressed in this commit - 806b792.